extensions/sql-format/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "sql-format",
+  "title": "Format SQL",
+  "description": "Format a SQL cluase use stanard SQL format",
+  "icon": "extension-icon.png",
+  "author": "tonngw",
+  "categories": [
+    "Data",
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Format Clipboard SQL",
+      "description": "Formats a SQL cluase string stored in clipboard and copies it back",
+      "mode": "no-view"
+    }
+  ],
+  "preferences": [
+    {
+      "default": false,
+      "description": "Paste the formatted JSON to an active app",
+      "label": "Automatically paste the formatted JSON",
+      "name": "autopaste",
+      "required": false,
+      "title": "Auto Paste",
+      "type": "checkbox"
+    },
+    {
+      "data": [
+        {
+          "title": "Default: 2",
+          "value": "2"
+        },
+        {
+          "title": "Spaces: 4",
+          "value": "4"
+        },
+        {
+          "title": "Tab: 8",
+          "value": "tab"
+        }
+      ],
+      "description": "Output sql tabWidth",
+      "name": "tabWidth",
+      "required": false,
+      "title": "tabWidth",
+      "type": "dropdown"
+    },
+    {
+      "data": [
+        {
+          "title": "Default",
+          "value": "preserve"
+        },
+        {
+          "title": "Upper",
+          "value": "upper"
+        },
+        {
+          "title": "Lower",
+          "value": "lower"
+        }
+      ],
+      "description": "Output sql keywordCase",
+      "name": "keywordCase",
+      "required": false,
+      "title": "keywordCase",
+      "type": "dropdown"
+    }
+  ],
+  "dependencies": {
+    "@raycast/api": "^1.81.2",
+    "@raycast/utils": "^1.16.3",
+    "sql-formatter": "^15.4.0"
+  },
+  "devDependencies": {
+    "@raycast/eslint-config": "^1.0.8",
+    "@types/node": "20.8.10",
+    "@types/react": "18.3.3",
```

Comment:
You can run either `npm run i @types/{node@22,react@18} -D` or you can even remove these dependencies `npm uninstall @types/{node, react}`